topics/go/algorithms/numbers/reverse/reverse.go

```diff
@@ -0,0 +1,24 @@
+package numbers
+
+// Reverse takes in a integer and returns a the integer in reverse order.
+func Reverse(num int) int {
+
+	// Set result to zero.
+	var result int
+
+	// Loop until num is not zero.
+	for num != 0 {
+
+		// Get the last digit from num.
```

Comment:
// Perform a modulus operation to get the last digit from the value set in num. 
// https://www.geeksforgeeks.org/find-first-last-digits-number/